test/e2e/common/node/lifecycle_hook.go

```diff
@@ -115,7 +115,7 @@ var _ = SIGDescribe("Container Lifecycle Hook", func() {
 				}, postStartWaitTimeout, podCheckInterval).Should(gomega.BeNil())
 			}
 			ginkgo.By("delete the pod with lifecycle hook")
-			podClient.DeleteSync(ctx, podWithHook.Name, *metav1.NewDeleteOptions(15), e2epod.DefaultPodDeletionTimeout)
+			podClient.DeleteSync(ctx, podWithHook.Name, *metav1.NewDeleteOptions(15), f.Timeouts.PodDelete)
```

Comment:
Can you please call out what is the difference between the two timeouts and highlight it in the PR text?

Reply:
I am guessing the new one is longer than the old one?

Reply:
Yes, 

https://github.com/kubernetes/kubernetes/blob/35d098aaa0a912171c46345e2c0a7b54d6046ef4/test/e2e/framework/pod/pod_client.go#L47

https://github.com/kubernetes/kubernetes/blob/35d098aaa0a912171c46345e2c0a7b54d6046ef4/test/e2e/framework/timeouts.go#L26

Reply:
please add this info to the PR text

Reply:
Done